scripts/analysis.py

```diff
@@ -56,188 +25,85 @@ def get_agent_ids(cursor, facility):
-    list: list
-        list that contains a column with numbers
-    column_index: int
-        index for the column to be summed
-
-    Returns
-    -------
-    int
-        summation of all the values in the array column
-    """
-    sum = 0
-    for ar in list:
-        sum += ar[column_index]
-
-    return sum
-
-
-def isotope_calc(wasteid_array, snf_inventory, cursor):
-    """ Calculates isotope mass using mass fraction in compositions table.
-
-        Fetches all compositions from compositions table.
-        Compositions table has the following format:
-            SimId / QualId / NucId / MassFrac
-        Then sees if the qualid matches, and if it does, multiplies
-        the mass fraction by the snf_inventory.
-
-    Parameters
-    ---------
-    wasteid_list: list
-        list of qualid of wastes
-    snf_inventory: float
-        total mass of snf [kg]
-    cursor: cursor
-        cursor for sqlite3
-
-    Returns
-    -------
-    nuclide_inven: list
-        inventory of individual nuclides.
-    """
-
-    # Get compositions of different waste
-    # SimId / QualId / NucId / MassFrac
-    cur = cursor
-    comps = cur.execute('select * from compositions').fetchall()
-    total_snf_mass = get_sum(snf_inventory, 0)
-
-    nuclide_inven = 'total snf inventory = ' + str(total_snf_mass) + 'kg \n'
-    nuclides = []
-    mass_of_nuclides = []
-    # if the 'qualid's match,
-    # the nuclide quantity and calculated and displayed.
-    for comp in comps:
-        for num in snf_inventory:
-            inv_qualid = num[1]
-            comp_qualid = comp[1]
-            if inv_qualid == comp_qualid:
-                comp_tot_mass = num[0]
-                mass_frac = comp[3]
-                nuclide_quantity = comp_tot_mass * mass_frac
-                nucid = comp[2]
-                nuclide_name = nucid
-                nuclides.append(nuclide_name)
-                mass_of_nuclides.append(nuclide_quantity)
-    return sum_nuclide_to_dict(nuclides, mass_of_nuclides)
-
-
-def sum_nuclide_to_dict(nuclides, nuclides_mass):
-    """takes a nuclide set and returns a dictionary with the masses of each nuclide
-
-    Parameters
-    ----------
-    nuclides: list
-        list of nuclides in the waste
-    nuclides_mass: list
-        list of nuclides' mass
-
-    Returns
-    -------
-    dict
-        dictionary of nuclide name and mass
-    """
-
-    nuclide_set = set(nuclides)
-    mass_dict = collections.OrderedDict()
-
-    for nuclide in nuclide_set:1
-        temp_nuclide_sum = 0
-        indeces = [i for i, x in enumerate(nuclides) if x == nuclide]
-        for index in indeces:
-            temp_nuclide_sum += nuclides_mass[index]
-        mass_dict[str(nuclide)] = temp_nuclide_sum
-
-    return mass_dict
+    return query
 
 
 def get_sim_time_duration(cursor):
-    """ Returns simulation time and duration of the simulation
+    """ Returns simulation start year, month, duration and
+    timesteps (in numpy linspace).
```

Comment:
if it does all of these things and not just the duration, why is the function called "get_sim_time_duration" ? Perhaps it should be get_timesteps ?